@viewplace/plot.m

```diff
@@ -65,7 +65,9 @@
 		sdstr = get(obj,'SessionDirs');
 		cwd = pwd;
 		sidx1 = sidx(1) - 1;
-		cd(sdstr{sidx1})
+% 		cd(sdstr{sidx1})
+        path = strrep(sdstr{sidx1},'/Users/shihcheng/afp','/');
```

Comment:
I wasn't sure if I should meddle with the @viewplace/plot (placeselect, create placeselect) because I only changed it quickly to try running the code on workstation18.

Reply:
Should I remove the changes or change the placeselect and createplaceselect code?

Reply:
I think we should try to find a way to make it more general so it will work for anyone trying to run the code.